### DISCUSSION TITLE
Replace Fauxton with custom app

### DISCUSSION THREAD

**Discussion by VladimirCores** [apache/couchdb#4481] [General]
Is there any way of how I could replace Fauxton with my custom UI?

**Comment by big-r81** [comment]
You could place your files in `couchdb/share/www`. This is the directory where all Fauxton files are stored.

**Reply by VladimirCores** [reply]
And what about path, could it be changed to something other that `/_utils`?

**Reply by big-r81** [reply]
No, `/_utils` is static. Fauxton is a standalone application and not coupled to CouchDB. It's only loaded if you call `/_utils`.
